code/tasks/FTPageTypeCreatorTask.php

```diff
@@ -78,17 +82,17 @@ private function getExistingClassNames($dir)
         $files = $this->finder
             ->in($dir)
             ->files()
-            ->name('*Page.php');
+            ->name('GeneratedPageType*.php');
 
         return array_map(function ($file) {
             /* @var SplFileInfo $file */
-            return $file->getBasename();
+            return basename($file->getBasename(), '.php');
         }, iterator_to_array($files));
     }
 
-    private function generateClassName()
+    private function generateClassName($counter)
     {
-        return ucfirst($this->faker->word . 'Page');
+        return 'GeneratedPageType' . $counter;
```

Comment:
This was the only spot `faker` was used. Can we remove `$this->faker` and `use Faker\Factory;`?